py_gtfs_rt_ingestion/tests/test_py_gtfs_rt_ingestion.py

```diff
@@ -0,0 +1,5 @@
+from py_gtfs_rt_ingestion import __version__
```

Comment:
super basic initial test that just checks version information.

as the library is built, we will add tests here that do things like create temp parquet files to read and make assertions on.